api/client/swarm/opts.go

```diff
@@ -86,40 +88,35 @@ func NewListenAddrOption() NodeAddrOption {
 
 // AutoAcceptOption is a value type for auto-accept policy
 type AutoAcceptOption struct {
-	values map[string]bool
+	values map[string]struct{}
 }
 
 // String prints a string representation of this option
 func (o *AutoAcceptOption) String() string {
 	keys := []string{}
-	for key, value := range o.values {
-		keys = append(keys, fmt.Sprintf("%s=%v", strings.ToLower(key), value))
+	for key := range o.values {
+		keys = append(keys, fmt.Sprintf("%s=%v", strings.ToLower(key), true))
```

Comment:
Ye I thought that too, maybe just for readability / clarity (for repr), although ye, not strictly required as it's true if it exists by default. Happy to change but think it's clearer to stay. No strong preference either way though.